webapp/src/pages/ResponsePage/ResponsePage.js

```diff
@@ -44,13 +46,14 @@ class ResponsePage extends Component {
             console.log(responses);
             this.setState({
                 eventLanguages: responses[0].event ? Object.keys(responses[0].event.name) : null,
+                event_type: responses[0].event.event_type,
                 applicationForm: responses[1].formSpec,
                 applicationData: responses[2].detail,
                 tagList: responses[3].tags,
                 reviewers: responses[4].reviewers,
                 error: responses[0].error || responses[1].error || responses[2].error || responses[3].error || responses[4].error,
-                isLoading: false
             }, this.handleData);
+            this.getOutcome();
```

Comment:
You'll still have the same issue here where you're trying to access this.state.applicationData inside getOutcome with no guarantee that it has been set yet by setState. So you'll need to call this where this.handleData is being called. You can update line 55 to be something like:
`}, ()=> {this.handleData(); this.getOutcome();}`  (with better formatting of course :) )